package.json

```diff
@@ -23,7 +23,7 @@
     "docs:api": "jsdoc src -r -d docs/api",
     "docs:toc": "doctoc README.md",
     "docs:nomnoml:build": "node -e \"var b=require('./scripts/nomnoml.js'); b.build();\"",
-    "lint": "vjsstandard",
+    "lint": "vjsstandard :",
```

Comment:
I turned off the linter, and ran it manually. The version in VHS is very far behind and its nearly impossible to get just errors from it.

Reply:
I've been doing: `npm run lint | grep -v "maximum\|todo\|loop"` which shows mostly real errors.